metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java

```diff
@@ -40,23 +41,12 @@ public void getMetadataJsonReturnsCreatePublicationRequest()
         URI uri = prepareWebServerAndReturnUriToMetdata(TEST_REVIEW_PAPER_HTML);
         Optional<CreatePublicationRequest> request = metadataService.getCreatePublicationRequest(uri);
 
-        assertKnownMetadataMappingsFromReviewPaperHtml(request.get());
+        File file = new File(TEST_REVIEW_PAPER_JSON);
```

Comment:
After the fix we made in commons, the following should work
```
String json = IoUtils.stringFromResources(Path.of(TEST_REVIEW_PAPER_JSON));
```
if not you can use
```
//imported  IoUtils.*
String json = streamToString(inputStreamFromResources(TEST_REVIEW_PAPER_JSON));
```

Reply:
Based on the title of the PR, it seems that one of these tests should be parametrized for the different variants. This is just a gut feeling though.

Reply:
Agreed, there is a necessity to test a broad range of resources with a number of expected outcomes. I'm not sure that flat files is the optimal way for targeted testing, though.